### PR TITLE
Feature: Add percentage difference

### PR DESCRIPTION
**Pull request by dduart3**
Since this wasn't a big deal I decided to fork the project and create a branch with a solution for this in which I did the following:

- Renamed the Injectionhandler.ts file located in the src/ route to injectionhandler.ts because it was causing errors at the time of the webpack compilation not allowing to finish it, also for standardization of all the file names in that route
- Added a helper for the percentage difference calculation in the src/util/helperfunctions.ts file
- Finally I did an small refactor in the addBuffPrice function and added the percentage difference result next to the buff price
- Fixed a case where the reference price was 0 because of no listed items or no active buy orders on certain items causing inconsistencies in price calculations, now it should display N/A when this happens, could be changed to anything else

### CODE REVIEW

File: src/util/helperfunctions.ts [hunk 1]
```diff
@@ -21,4 +21,8 @@ export function handleSpecialStickerNames(name: string): string {
         return name.substring(0, name.lastIndexOf('|')) + ' ' + name.substring(name.lastIndexOf('|'), name.length);
     }
     return name;
+}
+
+export function calculateRelativePercentageDifference(a: number, b: number) {
```
Comment (GODrums) [inline src/util/helperfunctions.ts:26]: This function is currently not in use, correct? I would love to see it removed then.

**Review by GODrums — CHANGES_REQUESTED** [review]
Thank you for your contribution, great changes so far! 

As with many other features, I would love to keep this an optional setting, as screen clustering has been one of our main issues in the past development process. 
If you know how, feel free to create a new extension setting under the "PRICES" section in the CSFloat tab of the popup which has to be enabled for the percentages to show. Otherwise, I can also commit this change to your PR myself.

Furthermore please review my one code comment, although this seems basically ready to merge, great work!

**Comment by dduart3** [comment]
> Thank you for your contribution, great changes so far!
> 
> As with many other features, I would love to keep this an optional setting, as screen clustering has been one of our main issues in the past development process. If you know how, feel free to create a new extension setting under the "PRICES" section in the CSFloat tab of the popup which has to be enabled for the percentages to show. Otherwise, I can also commit this change to your PR myself.
> 
> Furthermore please review my one code comment, although this seems basically ready to merge, great work!

I was thinking about that honestly before but I forget about it while submitting the PR, sure I will do it rn

File: src/util/helperfunctions.ts (hunk 1 above)
Comment (dduart3) [reply src/util/helperfunctions.ts:26]: > This function is currently not in use, correct? I would love to see it removed then.

oh my bad, it was being used before and i got rid of it in the main function but forgot to delete from the helperfunctions

Comment (dduart3) [reply src/util/helperfunctions.ts]: done, removed the function

**Comment by dduart3** [comment]
> Thank you for your contribution, great changes so far!
> 
> As with many other features, I would love to keep this an optional setting, as screen clustering has been one of our main issues in the past development process. If you know how, feel free to create a new extension setting under the "PRICES" section in the CSFloat tab of the popup which has to be enabled for the percentages to show. Otherwise, I can also commit this change to your PR myself.
> 
> Furthermore please review my one code comment, although this seems basically ready to merge, great work!

did this and tested it already! feel free to modify it

**Review by GODrums — APPROVED** [review]
Thank you for integrating my suggestions. LGTM now!

**Pull request merged**